realsense2_camera/launch/rs_launch.py

```diff
@@ -59,6 +59,7 @@
                            {'name': 'depth_module.gain.2',         'default': '16', 'description': 'Initial value for hdr_merge filter'},
                            {'name': 'wait_for_device_timeout',      'default': '-1.', 'description': 'Timeout for waiting for device to connect (Seconds)'},
                            {'name': 'reconnect_timeout',            'default': '6.', 'description': 'Timeout(seconds) between consequtive reconnection attempts'},
+                           {'name': 'rgb_camera.power_line_frequency', 'default': '1', 'description': 'Disabled - 0; 50 Hz - 1; 60 Hz - 2'},
```

Comment:
the default value might differ between different models. e.g., for D400 series:
0.f - Disabled
1.f - 50hz
2.f - 60 Hz
3.f - Auto

I think it is better to keep this as a user choice to add this parameter to the launch file in order to make it override-able for him.
You can always set/get these params within the ROS2 API (rclcpp, rclpy) or via the ROS2 command lines in the terminal (e.g., ros2 param set /camera/camera rgb_camera.power_line_frequency 1)
In other RealSense models, these values and their order may differ.
@Nir-Az, what do you think ?

Reply:
I agree, according to WMF docs for example:
![image](https://github.com/IntelRealSense/realsense-ros/assets/64067618/b069028e-8893-4e40-9345-e0704f94acad)

So better to keep it to the user.